docs/user-guide/petset.md

```diff
@@ -0,0 +1,370 @@
+* As with all alpha/beta resources, it can be disable through the `--runtime-config` option passed to the apiserver, and in fact most likely will be disabled on hosted offerings of Kubernetes.
+* The only updatable field on a Pet Set is `replicas`
+* The storage for a given pet is provisioned by a [dynamic storage provisioner](https://github.com/kubernetes/kubernetes/blob/release-1.3/examples/experimental/persistent-volume-provisioning/README.md) based on the requested `storage class`. Note that dynamic volume provisioning is also currently in alpha.
+* Deleting the Pet Set  *will not*  delete all pets. You will either have to manually scale it down to 0 pets first, or delete the pets yourself.
+* Deleting and/or scaling a Pet Set down will *not* delete the volumes associated with the Pet Set. This is done to ensure safety first, your data is more valuable than an auto purge of all related Pet Set resources. Deleting the Persistent Volume Claims will result in a deletion of the associated volumes.
+* All Pet Sets currently require a "governing service", or a Service responsible for the network identity of the pets. The user is responsible for this Service.
+* Updating an existing Pet Set is currently a manual process, meaning you either need to deploy a new Pet Set with the new image version, or orphan Pets one by one, update their image, and join them back to the cluster.
+
+
+## Properties of a Pet
+
+The following examples demonstrate the properties of Pets.
+
+### Identity
+
+A minimal Pet Set might look like:
+
+{% include code.html language="yaml" file="petset.yaml" ghlink="/docs/user-guide/petset.yaml" %}
+
+For completeness it includes the `Service` definition. Lets create the Pet Set:
+
+```shell
+$ kubectl create -f petset.yaml
+service "nginx" created
+petset "nginx" deleted
+```
+
+The identity of a Pet maps to the following:
+
+#### Ordinal index
+you should see 2 pods with predictable names formatted thus: `$(petset name)-$(ordinal index assigned by petset controller)`
+
+```shell
+$ kubectl get po
+NAME      READY     STATUS    RESTARTS   AGE
+web-0     1/1       Running   0          10m
+web-1     1/1       Running   0          10m
+```
+
+#### Stable storage
+2 persistent volumes, one per pod. This is auto created by the Pet Set based on the `volumeTemplate` field
+
+```shell
+$ kubectl get pv
+NAME                                       CAPACITY   ACCESSMODES   STATUS    CLAIM               REASON    AGE
+pvc-90234946-3717-11e6-a46e-42010af00002   1Gi        RWO           Bound     default/www-web-0             11m
+pvc-902733c2-3717-11e6-a46e-42010af00002   1Gi        RWO           Bound     default/www-web-1             11m
+```
+
+#### Network identity
+The network identity has 2 parts.
+
+First, we created a headless Service that controls the domain within which we create Pets. The domain managed by this Service takes the form: `$(service name).$(namespace).svc.cluster.local`, where "cluster.local" is the [cluster domain]().
```

Comment:
I added a table to clarify the mapping of service name <-> pet DNS/hostname. 
Stackoverflow tells us that people change cluster.local.